Code/GraphMol/ChemTransforms/ChemTransforms.cpp

```diff
@@ -51,6 +51,18 @@ void updateSubMolConfs(const ROMol &mol, RWMol &res,
     res.addConformer(newConf, false);
   }
 }
+
+struct SideChainMapping {
+  int molIndex;
+  int coreIndex = -1;
+  bool useMatch;
```

Comment:
useMatch could be set false here, or coreIndex should be unset, you override it in the constructor anyway.

Reply:
I'll unset coreIndex